SyncApp/Controllers/ConfigrationsController.cs

```diff
@@ -95,6 +98,29 @@ public async Task<IActionResult> Edit(Configrations configrations)
             }
             return View(configrations);
         }
+ 
+        public async Task<decimal?> GetTaxFromShopify(long id)
+        {
+            try
+            { 
+                var countryService = new CountryService(StoreUrl, ApiSecret);
+                Country country = await countryService.GetAsync(id);
+                var tax = country.Tax;
+                var countryDB = _countriesLogic.GetCountry(id);
+                if (countryDB != null)
+                {
+                    countryDB.CountryTax = tax;
+                    _context.Update(countryDB); 
+                    await _context.SaveChangesAsync();     
+                }
+                return tax;
+            }
+            catch (Exception ex)
+            {
+                _log.Error($"Exception While Getting Taxes From Shopify: {JsonConvert.SerializeObject(ex)}");
```

Comment:
there is an issue with the log , try throwing an exception it will reach the log but it will not reach the throw ex.

Reply:
Done